iree/compiler/Dialect/HAL/Target/LLVM/LLVMAOTTarget.cpp

```diff
@@ -442,6 +446,7 @@ class LLVMAOTTargetBackend final : public TargetBackend {
         builder, debugDatabaseFilenameRef);
     iree_DyLibExecutableDef_debug_database_embedded_add(builder,
                                                         debugDatabaseRef);
+    iree_DyLibExecutableDef_address_sanitized_add(builder, options_.addressSanitizer);
```

Comment:
here too an enum would allow us to add other sanitizer modes

Reply:
Yes, here my new commit has options.sanitizerKind is int type, but sanitized_kind in flatbuffer schema is ubyte. Can I use sanitizerKind as char type instead of int? I think one byte is enough to represent sanitizer modes..